src/ExRender.hs

```diff
@@ -98,7 +98,9 @@ instance ExRenderPackage GenericPackageDescription where
             "\""]
 
         pkgDescr = packageDescription descr
-
+        hackageRev = fmap snd
```

Comment:
Hint: `fmap f . g` usually can be replaced with `f <$> g`

Reply:
That's just style preference.